### PR TITLE
Added ios section to settings.json, added basic tests

### PR DESCRIPTION
**Pull request by artemryazantsev**
- Modified settings.json (added sections for iOS devices/driver configuration)
- Added basic integration tests for iOS
- Added some screens (page objects)

PR related to [#5](https://github.com/aquality-automation/aquality-appium-java/issues/5) - only part of tests

### CODE REVIEW

File: src/test/resources/settings.json [hunk 1]
```diff
@@ -21,6 +21,15 @@
       "capabilities": {
         "deviceName": "Android Emulator"
       }
+    },
+    "ios": {
+      "applicationPath": "./src/test/resources/apps/TestApp.app.zip",
```
Comment (mialeska) [inline src/test/resources/settings.json]: please include this file into this PR

File: src/test/resources/settings.json [hunk 2]
```diff
@@ -21,6 +21,15 @@
...
+      "capabilities": {
+        "deviceName": "iOS Simulator",
+        "automationName": "XCUITest",
+        "platformName": "iOS",
```
Comment (mialeska) [inline src/test/resources/settings.json]: are all these attributes required?

Comment (artemryazantsev) [reply src/test/resources/settings.json]: yes, default driver is UIAutomation (should be fixed in Appium official document too).

File: src/test/resources/settings.json (hunk 1 above)
Comment (artemryazantsev) [reply src/test/resources/settings.json]: done

**Comment by artemryazantsev** [comment]
@DmitryBogatko please review

File: src/test/java/samples/ios/IOSBasicInteractionsTest.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        driver = (IOSDriver<?>) ApplicationManager.getApplication().getDriver();
+    }
+
+    @AfterClass
+    public void tearDown() {
+        ApplicationManager.getApplication().quit();
+    }
+
+    @Test
+    public void testSendKeysToInput() {
```
Comment (DmitryBogatko) [inline src/test/java/samples/ios/IOSBasicInteractionsTest.java]: please, rename all the tests to patter like:
testShouldBePossibleToSendTextToElement()

do we really need MainScreen for such tests? Can we use elements factory directly here?

File: src/test/java/samples/ios/IOSBasicInteractionsTest.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        Assert.assertEquals(twoItemsAlertScreen.getTitle(), "Cool title");
+        twoItemsAlertScreen.acceptAlert();
+    }
+
+    @Test
+    public void testAcceptAlertViaDriver() {
+        MainScreen mainScreen = new MainScreen();
+        mainScreen.tapShowAlert();
+        TwoItemsAlertScreen twoItemsAlertScreen = new TwoItemsAlertScreen();
+        Assert.assertTrue(twoItemsAlertScreen.isDisplayed(), "Alert wasn't opened");
```
Comment (DmitryBogatko) [inline src/test/java/samples/ios/IOSBasicInteractionsTest.java]: why do we verify the same twice?

**Pull request closed without merging**